### DISCUSSION TITLE
discuss connecting mssql 2008 r2

### DISCUSSION THREAD

**Discussion by alavuvu** [catfan/Medoo#1100] [Q&A]
As the title says, I tried connecting to mssql 2014 and was able to use all the functions such as has , select , insert , update , ..., but the get function cannot be used on sql 2008 r2, but can still be used on sql 2014 . Has anyone encountered this problem, and does anyone know how to fix it to use the get function?

**Comment by catfan** [comment]
What is the actual problem on MSSQL 2008 for `get()`?

Anyway, it's 2023 year today and it's time to update.
